### ISSUE TITLE
How can I contribute ? 

### ISSUE THREAD

**Issue by NixonNelson** [minexew/TempleOS#8]
How Can i contribute to TempleOS ,
I am a Computer Science Student  ,I know a basic C, C++,Java .
How can I help ? @minexew

**Comment by NixonNelson** [comment]
> How Can i contribute to TempleOS ,
> I am a Computer Science Student ,I know a basic C, C++,Java .
> How can I help ?

**Comment by Midnoclose** [comment]
This is a mirror of TempleOS. You cannot contribute directly. If you know C, I recommend learning the dialect known as HolyC, which is used by TempleOS, and making a fork for yourself.

**Comment by minexew** [comment]
Indeed, the Temple is complete.

There are ways to contribute, though. You can write programs for TempleOS to increase its usefulness. Another thing you can do is help with finding and catalogizing historical versions and/or documenting the history as well as technical internals of the system.

**Comment by ghbjklhv** [comment]
> This is a mirror of TempleOS

@Midnoclose Where is the main repository? Is this not the main code repository?

**Comment by Midnoclose** [comment]
@ghbjklhv There is no main repo. The original source is from Terry’s website: https://templeos.org/Downloads/

**Comment by KeplerIO** [comment]
> @ghbjklhv There is no main repo. The original source is from Terry’s website: https://templeos.org/Downloads/

The source dir that is available from the website looks nothing like this repo. Is that because the source is only available in it's pure form after being booted into the OS?

**Comment by minexew** [comment]
@KeplerIO That's correct. You would download an ISO file, boot it, and then browse the source code from within TempleOS.

What you see in this repository is the extracted contents of that ISO.

**Comment by nhcfk** [comment]
I've acquired a certain greenish hue luminescence since compiling this. Is that normal?

**Comment by cloud11665** [comment]
No, they are after you. hide.

**Comment by Xyncgas** [comment]
There is no such thing as mental illness, they are but scientific approach to erase people's conscience. We're supposed to be in this world not of this world, taking the inability to understand a person to the next level as to judge them for what they do instead of minding your own business, I hear the devil at work whenever someone says 'mental illness'

For example someone said this guy first had bipolar, but it's just a way of life to search up and down for answers, so what the moon drives people crazy and we still look at it, god told us not to eat the apple and we still ate it, we're supposed to not wear any cloths now we have to put up with the judgement from the society and the hopes of others for us to have shame still we are trying to do what's right. If he had started scientology he would've been protected. How come everytimes the fed does something good for the country they hunt somebody down or manipulate the situations, instead of come to people who are good and give them an ice cream and some smoke and one good old hand shake

If he had joined https://kiwifarms.net/, he would've found there's a lot schizo or bipolar autistic people in this world, he's just a retard like the rest of us and that's ok
